docs/_spec/03-types.md

```diff
@@ -1012,7 +1012,11 @@ We define `memberType(´T´, ´id´, ´p´)` as follows:
   - If ´m´ is not defined, the result is undefined.
   - If ´m´ is a class declaration, the result is a class result with class ´m´.
   - If ´m´ is a term definition in class ´D´ with declared type ´U´, the result is a term result with underlying type [`asSeenFrom`](#as-seen-from)`(´U´, ´D´, ´p´)` and stable flag true if and only if ´m´ is stable.
-  - If ´m´ is a type member definition in class ´D´ with declared type definition ´U´, the result is a type result with underlying type definition [`asSeenFrom`](#as-seen-from)`(´U´, ´D´, ´p´)`.
+  - If ´m´ is a type member definition in class ´D´, the result is a type result with underlying type definition [`asSeenFrom`](#as-seen-from)`(´U´, ´D´, ´p´)` where ´U´ is defined as follows:
+      - If ´m´ is an opaque type alias member definition with declared definition ´>: L <: H = V´, then
+          - ´U´ is ´= V´ if `´p = D.´this` or if we are computing `memberType` in a [_transparent mode_](#type-erasure),
```

Comment:
is it more accurate to say if `D.this` is a subprefix of `p` ?

Reply:
or is another rule needed for the nested scopes?

Reply:
Even if you are within a class `E` that is nested inside `D`, if you refer to a member declared at the level of `D`, the prefix is `D.this`.

Reply:
ok